### DISCUSSION TITLE
Space Parts dataset

### DISCUSSION THREAD

**Discussion by salilcbi** [TabularEditor/TabularEditor3#1374] [Learn]
Is the Space Parts dataset available for non-commercial usage in csv, parquet or delta formats or is it only bundled in the pbix files?

**Comment by otykier** [comment]
You can get the entire dataset in CSV/parquet from [here](https://cdn.tabulareditor.com/files/spaceparts-co.zip) (847 MiB).

We also have individual files for fact/dimension tables available:

- Facts:
  - [orders.parquet](https://cdn.tabulareditor.com/files/spaceparts-co/fact/orders.parquet) (368 MiB)
  - [invoices.parquet](https://cdn.tabulareditor.com/files/spaceparts-co/fact/invoices.parquet) (458 MiB)
  - [budget.parquet](https://cdn.tabulareditor.com/files/spaceparts-co/fact/budget.parquet) (21 MiB)
  - [forecast.csv](https://cdn.tabulareditor.com/files/spaceparts-co/fact/forecast.csv) (220 KiB)
- Dimensions:
  - [products.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/products.csv) (61 MiB)
  - [brands.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/brands.csv) (1.5 KiB)
  - [budget-rate.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/budget-rate.csv) (0.5 KiB)
  - [customers.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/customers.csv) (555 KiB)
  - [employees.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/employees.csv) (142 KiB)
  - [exchange-rate.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/exchange-rate.csv) (5 MiB)
  - [invoice-doctype.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/invoice-doctype.csv) (0.2 KiB)
  - [order-doctype.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/order-doctype.csv) (0.2 KiB)
  - [order-status.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/order-status.csv) (0.3 KiB)
  - [regions.csv](https://cdn.tabulareditor.com/files/spaceparts-co/dim/regions.csv) (23.6 KiB)

Please note the **[license terms](https://tabulareditor.com/terms)** for use of these files, specifically section 5 which is quoted below:

> **5. Intellectual Property:**
> All content on www.tabulareditor.com, including Tabular Editor Learn course material; such as, but not limited to, developed datasets, models, documents, slides, images, audio, and video (the "Material"), is subject to copyright, trademark and other intellectual property law protection. Tabular Editor ApS exclusively owns all rights, title, and interest in and to all the Material.
>
> By using the Material, you acknowledge and consent to adhere to the terms and conditions below:
>
> **5.1 Personal and educational use:**
> You are granted permission to use the Material for personal learning and educational purposes directly associated with Tabular Editor Learn course. Further, we appreciate and encourage your interest in sharing knowledge and insights. You may therefore share screenshots or minor excerpts of the Material on social media, blogs and similar platforms for non-commercial purposes, provided that proper credit is given to Tabular Editor ApS as the copyright owner.
>
> **5.2 Non-commercial use:**
> Upon prior written permission by Tabular Editor ApS, you are entitled to use the Material, in whole or in part, for non-commercial purposes, such as presentations at community events (e.g., sessions at conferences, pre-cons, user groups) or similar events/activities, provided that no entrance fee or other compensation is received. In the event such permission is granted, proper credit must be given to Tabular Editor ApS as the copyright owner.
> 
> **5.3 Prohibited actions:**
You are explicitly prohibited from modifying, reproducing, and using the Material, in whole or in part, for any commercial purposes, e.g., consultancy work, paid classroom training, certifications or similar commercial activities.

**Comment by corycundy** [comment]
Hi @otykier, thanks for posting this. This will be useful for additional learning and testing on different types of models, using Lake houses, etc.

If trying to mimic some of the examples in the training, it would be helpful to have the View definitions from the SQL Server database. Any logic, as minimal as it might be, would help confirm any variances if trying to compare to an example in the training.

Would it be possible to script the View Definitions and include those either in the zip or as a separate file download?

Cory Cundy
